community/Makefile

```diff
@@ -10,11 +10,9 @@ reset-docs:
 	git checkout HEAD -- ./sig-list.md ./sig-*/README.md ./wg-*/README.md
 
 generate:
```

Comment:
What about taking the opportunity to change the target name to make it more clear that this is to generate SIG-related files? (to make it easier to differentiate from the label-related generation below). Something like `sig-sync` (similar to `label-sync` that you have below), or `sig-generate` (if we want to keep the _generate_ part)?